CMakeLists.txt

```diff
@@ -16,26 +26,21 @@ option(USE_BOOST_REGEX "Replace std::regex with Boost.Regex" OFF)
 # C++ standard
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
-if(NOT CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 11)
-	set(CMAKE_CXX_STANDARD 11)
+if(NOT CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 17)
+    set(CMAKE_CXX_STANDARD 17)
 endif()
 
 #============================================================================
 # Sources & headers
 #============================================================================
 set(docopt_SOURCES docopt.cpp)
-set(docopt_HEADERS
-		docopt.h
-		docopt_private.h
-		docopt_util.h
-		docopt_value.h
-		)
+set(docopt_HEADERS docopt.h docopt_private.h docopt_util.h docopt_value.h)
```

Comment:
Unnecessary whitespace change; remove

Reply:
I use `cmake-format -i CMakeLists.txt` to format the hole file!

Reply:
Ah, I misunderstood that.  I think that collapsing the header list is a pessimization and I appreciate filelists like this being one line per file to make diffs simple.

Reply:
done